ports/cortex_a12/gnu/example_build/sample_threadx.ld

```diff
@@ -109,7 +109,7 @@ SECTIONS
   .eh_frame_hdr : { *(.eh_frame_hdr) }
   /* Adjust the address for the data segment.  We want to adjust up to
      the same address within the page on the next page up.  */
-  . = ALIGN(256) + (. & (256 - 1));
+  . = 0x2E000000;
```

Comment:
Seems it's a fixed address, is that fit for all OEM chips or just for specific one?

Reply:
yes, its fixed address:
Refer to:
https://developer.arm.com/documentation/100964/1123/Versatile-Express-Model/VE-memory-map-for-Cortex-A-series?lang=en